lockbox-ios/Action/DataStoreAction.swift

```diff
@@ -273,6 +292,26 @@ extension DataStoreActionHandler {
                 .disposed(by: self.disposeBag)
     }
 
+    private func _touch(_ item: Item) {
+        self.openSubject.take(1)
+                .flatMap { _ in
+                    self.checkState()
+                }
+                .flatMap { _ -> Single<Any> in
+                    if item.id == nil {
+                        throw DataStoreError.NoIDPassed
+                    }
+
+                    let jsonItem = try self.parser.jsonStringFromItem(item)
+
+                    return self.webView.evaluateJavaScript("\(self.dataStoreName).touch(\(jsonItem))")
```

Comment:
I… don't… understand… this code. 

Doesn't `evaluateJavascript` return `void`? Oh, god it, Rx monkey patches.